docs/Dockerfile

```diff
@@ -21,7 +21,7 @@ RUN adduser --system --uid 1001 nextjs
 
 
 # Copy the bundled code from the build stage to the production image
-COPY --from=build --chown=nextjs:nodejs /app/.next/standalone ./
+# COPY --from=build --chown=nextjs:nodejs /app/.next/standalone ./
```

Comment:
I'm not a fan of commenting out code, if it isn't needed it should just be removed.